Code/Framework/AzFramework/Platform/Common/UnixLike/AzFramework/Device/DeviceAttributesCommon_UnixLike.cpp

```diff
@@ -29,14 +31,25 @@ namespace AzFramework
         {
             char buffer[256] = { 0 };
             AZ::u64 memTotalKiB = 0;
+            constexpr AZStd::string_view MemTotalKey = "MemTotal:";
             while (fgets(buffer, sizeof(buffer), f))
             {
-                if (azsscanf(buffer, "MemTotal: %d", &memTotalKiB) && memTotalKiB > 0)
+                const AZStd::string_view bufferView(buffer);
+                if(size_t offset = bufferView.find(MemTotalKey); offset != AZStd::string_view::npos)
                 {
-                    // meminfo displays memory in KiB (kilobytes, base 1024) 
-                    // convert to GiB
-                    constexpr double KiBtoGiB = 1024.f * 1024.f;
-                    m_value = aznumeric_cast<float>(static_cast<double>(memTotalKiB) / KiBtoGiB);
+                    // skip spaces because from_chars does not support non numeric string prefixes 
+                    // non-numeric string postfixes are OK
+                    auto iter = bufferView.begin() + MemTotalKey.size();
+                    while(iter != bufferView.end() && *iter == ' ')

```

Comment:
Shouldn't this check `AZStd::isspace(*iter)`? in case there is a tab in the file.

Reply:
Good call, it looks like [top ](https://github.com/mmalecki/procps/blob/master/proc/sysinfo.c#L608C22-L608C29)uses `strtoul` which ignores whitespace.